metricflow/test/fixtures/sql_client_fixtures.py

```diff
@@ -74,7 +74,8 @@ def __configure_test_env_from_url(url: str, password: str, schema: str) -> sqlal
 
 def __configure_bigquery_env_from_credential_string(password: str, schema: str) -> None:
     credential_string = password.replace("'", "")
-    credentials = json.loads(credential_string)
+    # `strict=False` required to work with BQ password characters.
+    credentials = json.loads(credential_string, strict=False)
```

Comment:
Interesting, the CI credential string doesn't need this. I can't remember what kind of escaping I did to get it working, though. Must be something to do with the nested JSON. Anyway everything seems fine so 🤷